python/cugraph/cugraph/structure/graph_implementation/simpleGraph.py

```diff
@@ -1306,6 +1335,14 @@ def neighbors(self, n):
             n = node[0]
 
         df = self.edgelist.edgelist_df
+
+        vertex_col_name = [simpleGraphImpl.srcCol, simpleGraphImpl.dstCol]
+
+        if not self.properties.multi_edge:
+            # Drop parallel edges for non MultiGraph
+            # FIXME: Drop multi edges with the CAPI instead.
+            df = df.groupby(by=[*vertex_col_name], as_index=False).min()
```

Comment:
Same question here , assuming this a cuDF dataframe.

Reply:
Yes it is a cuDF dataframe

Reply:
Thanks